core/trie2/databasetest.go

```diff
@@ -26,7 +26,9 @@ func newTestNodeReader(id trieutils.TrieID, nodes []*trienode.MergeNodeSet, db d
 	return &testNodeReader{id: id, nodes: nodes, db: db, scheme: scheme}
 }
 
-func (n *testNodeReader) Node(owner *felt.Felt, path *trieutils.Path, hash *felt.Felt, isLeaf bool) ([]byte, error) {
+func (n *testNodeReader) Node(
+	owner *felt.Address, path *trieutils.Path, hash *felt.Felt, isLeaf bool,
```

Comment:
Can we split these args into multiple lines?

Reply:
done